src/crypto/webcrypto-browser.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+module.exports = function getWebCrypto () {
+  if (typeof window !== 'undefined') {
```

Comment:
No, we don't want to know if we are in node, we explicitly want to know if there is a `window` object available.